ibm/config.go

```diff
@@ -594,91 +594,368 @@ func (sess clientSession) IAMNamespaceAPI() (*ns.IbmCloudFunctionsNamespaceAPIV1
+			URL:            sess.cisDNSRecordsClient.Service.Options.URL,
+			Crn:            core.StringPtr(""),
+			ZoneIdentifier: core.StringPtr(""),
+			Authenticator:  sess.cisDNSRecordsClient.Service.Options.Authenticator,
+		}
+		cisDNSRecordsClient, cisDNSErr := cisdnsrecordsv1.NewDnsRecordsV1(cisDNSRecordsOpt)
+		if cisDNSErr != nil {
+			return nil, fmt.Errorf(
+				"Error occured while configuring CIS DNS client: %s", cisDNSErr)
+		}
+		return cisDNSRecordsClient, nil
+	}
 	return sess.cisDNSRecordsClient, sess.cisDNSErr
 }
 
 // CIS GLB Pool
 func (sess clientSession) CisGLBPoolClientSession() (*cisglbpoolv0.GlobalLoadBalancerPoolsV0, error) {
+	if sess.cisGLBPoolErr == nil {
+		cisGLBPoolOpt := &cisglbpoolv0.GlobalLoadBalancerPoolsV0Options{
+			URL:           sess.cisGLBPoolClient.Service.Options.URL,
+			Crn:           core.StringPtr(""),
+			Authenticator: sess.cisGLBPoolClient.Service.Options.Authenticator,
+		}
+		cisGLBPoolClient, cisGLBPoolErr :=
+			cisglbpoolv0.NewGlobalLoadBalancerPoolsV0(cisGLBPoolOpt)
+		if cisGLBPoolErr != nil {
+			return nil, fmt.Errorf(
+				"Error occured while configuring CIS GLB Pool client: %s",
+				cisGLBPoolErr)
+		}
+		return cisGLBPoolClient, nil
+	}
 	return sess.cisGLBPoolClient, sess.cisGLBPoolErr
 }
 
 // CIS GLB
 func (sess clientSession) CisGLBClientSession() (*cisglbv1.GlobalLoadBalancerV1, error) {
+	if sess.cisGLBErr == nil {
+		cisGLBOpt := &cisglbv1.GlobalLoadBalancerV1Options{
+			URL:            sess.cisGLBClient.Service.Options.URL,
+			Authenticator:  sess.cisGLBClient.Service.Options.Authenticator,
+			Crn:            core.StringPtr(""),
+			ZoneIdentifier: core.StringPtr(""),
+		}
+		cisGLBClient, cisGLBErr := cisglbv1.NewGlobalLoadBalancerV1(cisGLBOpt)
+		if cisGLBErr != nil {
+			return nil, fmt.Errorf(
+				"Error occured while configuring CIS GLB client: %s",
+				cisGLBErr)
+		}
+		return cisGLBClient, nil
+	}
 	return sess.cisGLBClient, sess.cisGLBErr
 }
 
 // CIS GLB Health Check/Monitor
 func (sess clientSession) CisGLBHealthCheckClientSession() (*cisglbhealthcheckv1.GlobalLoadBalancerMonitorV1, error) {
+	if sess.cisGLBHealthCheckErr == nil {
+		cisGLBHealthCheckOpt := &cisglbhealthcheckv1.GlobalLoadBalancerMonitorV1Options{
+			URL:           sess.cisGLBHealthCheckClient.Service.Options.URL,
+			Crn:           core.StringPtr(""),
+			Authenticator: sess.cisGLBHealthCheckClient.Service.Options.Authenticator,
+		}
+		cisGLBHealthCheckClient, cisGLBHealthCheckErr :=
+			cisglbhealthcheckv1.NewGlobalLoadBalancerMonitorV1(cisGLBHealthCheckOpt)
+		if cisGLBHealthCheckErr != nil {
+			return nil, fmt.Errorf(
+				"Error occured while configuring CIS GLB Health Check client: %s",
+				cisGLBHealthCheckErr)
+		}
+		return cisGLBHealthCheckClient, nil
+	}
 	return sess.cisGLBHealthCheckClient, sess.cisGLBHealthCheckErr
 }
 
 // CIS Zone Rate Limits
 func (sess clientSession) CisRLClientSession() (*cisratelimitv1.ZoneRateLimitsV1, error) {
+	if sess.cisRLErr == nil {
```

Comment:
Same here:

```go
if sess.cisRLErr != nil {
	return sess.cisRLClient, sess.cisRLErr
}

cisRLOpt := &cisratelimitv1.ZoneRateLimitsV1Options{
	URL:            sess.cisRLClient.Service.Options.URL,
	Crn:            core.StringPtr(""),
	ZoneIdentifier: core.StringPtr(""),
	Authenticator:  sess.cisRLClient.Service.Options.Authenticator,
}
cisRLClient, cisRLErr := cisratelimitv1.NewZoneRateLimitsV1(cisRLOpt)
if cisRLErr != nil {
	return nil, fmt.Errorf(
		"Error occured while cofiguring CIS Zone Rate Limit client: %s",
		cisRLErr)
}

return cisRLClient, nil

```

Reply:
I have upgraded sdk and regenerated cis sdk. Modified the code as per your suggession.